modules/ROOT/pages/config-add-proxy-whitelist.adoc

```diff
@@ -27,3 +27,53 @@ The following example defines regular expressions that allow requests to be made
 ----
 kubectl delete pod -n api-manager -l component=api-platform-web
 ----
+
+NOTE: This method is currently working from version 3.2.0 to version 3.2.1. From version 3.2.2 and any future versions, refer to Trusted Domains.
```

Comment:
It is not clear what "this method" refers to. Is it all of the preceding steps or just the last one?